azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/AudioSessionManager.kt

```diff
@@ -38,6 +38,7 @@ internal class AudioSessionManager(
     private val audioManager by lazy { context.getSystemService(Context.AUDIO_SERVICE) as AudioManager }
     private var bluetoothAudioProxy: BluetoothHeadset? = null
     private var initialized = false
+    private var isProfileProxyOpened = false
 
     private val isBluetoothScoAvailable
         get() = try {
```

Comment:
can you add enclosing brackets in line46 to fix Android studio warning

Reply:
Do you mean {} ? 
There's an enclosing one on line 47.

Reply:
changed as per discussion.